tensorflow/contrib/model_pruning/python/pruning.py

```diff
@@ -520,7 +520,7 @@ def add_pruning_summaries(self):
       thresholds = get_thresholds()
       for index, mask in enumerate(masks):
         if not self._exists_in_do_not_prune_list(mask.name):
-          summary.scalar(mask.name + '/sparsity', nn_impl.zero_fraction(mask))
+          summary.scalar(mask.op.name + '/sparsity', nn_impl.zero_fraction(mask))
```

Comment:
This _looks_ right. 

This wouldn't be broken if there was a test for it, any chance you could add one with this change so that this stays fixed?

Reply:
I'm pretty novice to designing tests; how would you go about doing this? I looked at the tests in [pruning_test.py](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/model_pruning/python/pruning_test.py), but didn't get very far.